lit_gpt/config.py

```diff
@@ -24,6 +24,7 @@ class Config:
     rotary_percentage: float = 0.25
     parallel_residual: bool = True
     bias: bool = True
+    use_bos: bool = False
```

Comment:
I would prefer that we read this off the `tokenizer_config` instead of adding this setting to the model.